spec.html

```diff
@@ -36631,8 +36625,7 @@
               1. If replacing the |FunctionDeclaration| _f_ with a |VariableStatement| that has _F_ as a |BindingIdentifier| would not produce any Early Errors for _func_ and _F_ is not an element of BoundNames of _argumentsList_, then
                 1. NOTE A var binding for _F_ is only instantiated here if it is neither a VarDeclaredName, the name of a formal parameter, or another |FunctionDeclaration|.
                 1. If _instantiatedVarNames_ does not contain _F_, then
-                  1. Let _status_ be _varEnvRec_.CreateMutableBinding(_F_).
-                  1. Assert: _status_ is never an abrupt completion.
+                  1. Let _status_ be ! _varEnvRec_.CreateMutableBinding(_F_).
```

Comment:
this `_status_` is never used, should it be `Perform ! _varEnvRec_.CreateMutableBinding(_F_).`?